# dev-lang/python: cross compiled python installs wrong version of lib2to3/Grammar pickle.
# https://bugs.gentoo.org/704816

diff --git a/Lib/lib2to3/pgen2/driver.py b/Lib/lib2to3/pgen2/driver.py
index cbc58e759d..42a3f7448d 100644
--- a/Lib/lib2to3/pgen2/driver.py
+++ b/Lib/lib2to3/pgen2/driver.py
@@ -108,7 +108,7 @@ def _generate_pickle_name(gt):
     head, tail = os.path.splitext(gt)
     if tail == ".txt":
         tail = ""
-    return head + tail + ".".join(map(str, sys.version_info)) + ".pickle"
+    return head + tail + ".pickle"


 def load_grammar(gt="Grammar.txt", gp=None,
